### PR TITLE
Fix review excerpt

### PR DESCRIPTION
**Pull request by confusedsushi**
Currently the "File Changed" tab of a PR is somehow broken. This is also true for the current release 1.16.0.

When you are on the "File Changed" tab, and want to look at code excerpt before or after the code changes, the layout breaks. You can test this on try.gitea.io here: https://try.gitea.io/testnotexisting/magic_enum/pulls/2/files

The problem occurs for the unified view and for the split view.

Kind of the same problem was there for commenting a line of code, this was fixed in #18321 and #18403.

For consistency, I changed the solution of #18321, I removed the ``colspan`` and instead added a ``<td>``. The goal was to have code similarly with the split view.

Also the separator line in the split view was in the wrong column, this was fixed too.

### CODE REVIEW

**Comment by lunny** [comment]
Fix #18517?

**Comment by confusedsushi** [comment]
Yes, this is fixing #18516.

#18517 is something different.

**Review by zeripath — APPROVED** [review]
Sorry for another bug here. The diff code is remarkably complex and fiddly. 

I've fixed this so that the escaped warning triangle appears on the excerpted lines too.

**Comment by zeripath** [comment]
@silverwind I think this is now fixed. 

I don't understand why the colspan in the js has been switched to three separate columns in the js - because that is not related to these code changes but I guess it might just help with keeping the same rendering so I've left it.

**Comment by silverwind** [comment]
JS should probably not create DOM elements from scratch but clone existing ones and alter them. That way, a whole class of possible errors is eliminated :)

**Review by silverwind — APPROVED** [review]

**Comment by confusedsushi** [comment]
> @silverwind I think this is now fixed.
> 
> I don't understand why the colspan in the js has been switched to three separate columns in the js - because that is not related to these code changes but I guess it might just help with keeping the same rendering so I've left it.

I wanted to have the similarity to the table layout of other rows. There you have ``line-number-old | line-number-new`` it was before collapsed with the ``colspan``.

With your changes now, there is again one line number column missing.

**Comment by zeripath** [comment]
Show me

File: web_src/js/features/repo-issue.js [hunk 1]
```diff
@@ -505,7 +505,9 @@ export function initRepoPullRequestReview() {
             <td class="lines-type-marker"></td>
             <td class="add-comment-right"></td>
           ` : `
-            <td colspan="3" class="lines-num"></td>
+            <td class="lines-num"></td>
+            <td class="lines-escape"></td>
+            <td class="lines-type-marker"></td>
```
Comment (zeripath) [inline web_src/js/features/repo-issue.js]:
```suggestion
            <td class="lines-num"></td>
            <td class="lines-num"></td>
            <td class="lines-escape"></td>
```

**Comment by zeripath** [comment]
Please send backport

**Pull request merged**